### PR TITLE
pt-BR

### PR DESCRIPTION
**Pull request by caosdp-rs**
Supersedes #613

### CODE REVIEW

File: src/Language/pt-BR/Auth.php [hunk 1]
```diff
@@ -0,0 +1,100 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodeIgniter\Shield\Language\en;
```
Comment (datamweb) [inline src/Language/pt-BR/Auth.php:5]:
```suggestion
namespace CodeIgniter\Shield\Language\pt-BR;
```

File: tests/Language/AbstractTranslationTestCase.php [hunk 2]
```diff
@@ -66,7 +66,7 @@ abstract class AbstractTranslationTestCase extends TestCase
         //        NorwegianTranslationTest::class          => 'no',
         //        PolishTranslationTest::class             => 'pl',
         //        PortugueseTranslationTest::class         => 'pt',
-        //        BrazilianTranslationTest::class          => 'pt-BR',
+        BrazilianTranslationTest::class          => 'pt-BR',
```
Comment (datamweb) [inline tests/Language/AbstractTranslationTestCase.php:69]:
```suggestion
        BrazilianTranslationTest::class => 'pt-BR',
```

**Comment by datamweb** [comment]
Hello.
Thanks for the PR and welcome to Shield.

Please sign **all** your commits.
Screenshot 2023-02-01 181020

Also, we have the [error Code Style](https://github.com/codeigniter4/shield/actions/runs/4061340348/jobs/6994498274), to fix it, you can run the following command:

```console
composer cs-fix 
```

**Comment by datamweb** [comment]
Hi @kenjis, 
The question is, why have we used the **namespace** in the language file! I didn't see any explanation in the documentation about using the namespace for the languages file.
See https://codeigniter.com/user_guide/outgoing/localization.html#id6

**Comment by kenjis** [comment]
@datamweb Setting namespace in lang files is a mistake.

I sent PR https://github.com/codeigniter4/CodeIgniter4/pull/7210

File: src/Language/pt-BR/Auth.php [hunk 3]
```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace CodeIgniter\Shield\Language\pt-BR;
```
Comment (kenjis) [inline src/Language/pt-BR/Auth.php:6]: Please remove these lines.

**Comment by kenjis** [comment]
@caosdp-rs It seems you added commits that you didn't make.
Please run `git rebase` to update your PR branch.
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#updating-your-branch

**Comment by caosdp-rs** [comment]
> @caosdp-rs It seems you added commits that you didn't make. Please run `git rebase` to update your PR branch. See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#updating-your-branch

PS C:\candre\shield> git rebase        
Current branch develop is up to date.

**Comment by kenjis** [comment]
```
> git fetch upstream
> git switch develop
> git rebase upstream/develop
```

**Comment by kenjis** [comment]
There are 4 failing checks in GitHub Actions.
Please fix them.

And I would like you to `git rebase` not `git merge` if you can.

**Comment by caosdp-rs** [comment]
> There are 4 failing checks in GitHub Actions. Please fix them.
> 
> And I would like you to `git rebase` not `git merge` if you can.

PS C:\candre\shield> git rebase                 
error: cannot rebase: You have unstaged changes.
error: Please commit or stash them.

**Comment by kenjis** [comment]
```
> git fetch upstream
> git switch develop
> git rebase upstream/develop
```

**Comment by caosdp-rs** [comment]
> git rebase upstream/develop

PS C:\candre\shield> git remote add upstream https://github.com/codeigniter4/shield.git
PS C:\candre\shield> git fetch upstream
remote: Enumerating objects: 84, done.
remote: Counting objects: 100% (28/28), done.
remote: Compressing objects: 100% (11/11), done.
remote: Total 84 (delta 17), reused 18 (delta 17), pack-reused 56
Unpacking objects: 100% (84/84), 528.47 KiB | 1.06 MiB/s, done.
From https://github.com/codeigniter4/shield
 * [new branch]      develop       -> upstream/develop
 * [new branch]      gh-pages      -> upstream/gh-pages
 * [new branch]      master        -> upstream/master
 * [new tag]         v1.0.0-beta   -> v1.0.0-beta
 * [new tag]         v1.0.0-beta.2 -> v1.0.0-beta.2
 * [new tag]         v1.0.0-beta.3 -> v1.0.0-beta.3
PS C:\candre\shield> git switch develop
Already on 'develop'
M       tests/Language/BrazilianTranslationTest.php
Your branch is up to date with 'origin/develop'.
PS C:\candre\shield> git rebase upstream/develop
error: cannot rebase: You have unstaged changes.
error: Please commit or stash them.

**Comment by kenjis** [comment]
```
> git stash
> git rebase upstream/develop
```

If you want to get back your local changes:
```
> git stash pop
```

File: docs/authorization.md [hunk 4]
```diff
@@ -129,6 +129,23 @@ if (! $user->hasPermission('users.create')) {
 }
 ```
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
```
Comment (kenjis) [inline docs/authorization.md:134]: Please resolve conflicts.

File: docs/authorization.md [hunk 5]
```diff
@@ -129,6 +129,23 @@ if (! $user->hasPermission('users.create')) {
...
+```php
+public $filters = [
+    'group:admin,superadmin' => ['before' => ['admin/*']],
+    'permission:users.manage' => ['before' => ['admin/users/*']],
+];
+```
+
+>>>>>>> 39f1a58 (docs: make file paths bold)
+=======
+>>>>>>> 9981e4e (docs: remove unsupported code sample)
```
Comment (kenjis) [inline docs/authorization.md:148]: Please resolve conflicts.

**Pull request closed without merging**